cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -1320,50 +1320,54 @@ func TestControllerManagerAliases(t *testing.T) {
 }
 
 func TestWatchListClientFlagUsage(t *testing.T) {
-	assertWatchListClientFeatureDefaultValue(t)
-
 	fs := pflag.NewFlagSet("addflagstest", pflag.ContinueOnError)
 	s, _ := NewKubeControllerManagerOptions()
 	for _, f := range s.Flags([]string{""}, []string{""}, nil).FlagSets {
 		fs.AddFlagSet(f)
 	}
 
-	fgFlagName := "feature-gates"
-	fg := fs.Lookup(fgFlagName)
-	if fg == nil {
-		t.Fatalf("didn't find %q flag", fgFlagName)
-	}
-
-	expectedWatchListClientString := "WatchListClient=true|false (BETA - default=false)"
-	if !strings.Contains(fg.Usage, expectedWatchListClientString) {
-		t.Fatalf("%q flag doesn't contain the expected usage for %v feature gate.\nExpected = %v\nUsage = %v", fgFlagName, clientgofeaturegate.WatchListClient, expectedWatchListClientString, fg.Usage)
-	}
+	assertWatchListClientFeatureDefaultValue(t)
+	assertWatchListCommandLineDefaultValue(t, fs)
```

Comment:
Can you move this to a separate PR, at least to some extent it seems like a cleanup.

Reply:
sure, opened https://github.com/kubernetes/kubernetes/pull/125723